website/views/shared/profiles/stats.jade

```diff
@@ -31,30 +31,30 @@ unless mobile
-            span.hint(popover-title=env.t('levelBonus'), popover-trigger='mouseenter', popover-placement='top', popover=env.t('levelBonusText'))=env.t('level')
-            |: {{Math.ceil((Math.min(profile.stats.lvl - 1, 100)) / 2)}}&nbsp;
-          li(ng-show='g[e.weapon].#{k} + g[e.armor].#{k} + g[e.head].#{k} + g[e.shield].#{k} > 0')
-            span.hint(popover-title=env.t('equipment'), popover-trigger='mouseenter', popover-placement='top', popover=env.t('equipmentBonusText'))=env.t('equipment')
-            |: {{g[e.weapon].#{k} + g[e.armor].#{k} + g[e.head].#{k} + g[e.shield].#{k} || 0}}&nbsp;
-          li(ng-show='profile._statsComputed.#{k} - profile.stats.buffs.#{k} - Math.ceil((Math.min(profile.stats.lvl - 1, 100)) / 2) - g[e.weapon].#{k} - g[e.armor].#{k} - g[e.head].#{k} - g[e.shield].#{k} - profile.stats.#{k} > 0')
-            span.hint(popover-title=env.t('classBonus'), popover-trigger='mouseenter', popover-placement='top', popover=env.t('classBonusText'))=env.t('classEquipBonus')
-            |: {{profile._statsComputed.#{k} - profile.stats.buffs.#{k} - Math.ceil((Math.min(profile.stats.lvl - 1,100)) / 2) - g[e.weapon].#{k} - g[e.armor].#{k} - g[e.head].#{k} - g[e.shield].#{k} - profile.stats.#{k}}}&nbsp;
-          li(ng-show='profile.stats.#{k} > 0')
-            span.hint(popover-title=env.t('allocatedPoints'), popover-trigger='mouseenter', popover-placement='top', popover=env.t('allocatedPointsText'))=env.t('allocated')
-            |: {{profile.stats.#{k} || 0}}&nbsp;
-          li(ng-show='profile.stats.buffs.#{k} > 0')
-            span.hint(popover-title=env.t('buffs'), popover-trigger='mouseenter', popover-placement='top', popover=env.t('buffsText'))=env.t('buffs')
-            |: {{profile.stats.buffs.#{k} || 0}}&nbsp;
+  //each v,k in { str: {title:"strength",popover:'strengthText'},int: {title:"intelligence",popover:'intText'},con: {title:"constitution",popover:'conText'},per: {title:"perception",popover:'perText'} }
+  tr(ng-repeat='(k,v) in { str: {title:"strength",popover:"strengthText"},int: {title:"intelligence",popover:"intText"},con: {title:"constitution",popover:"conText"},per: {title:"perception",popover:"perText"} }', ng-init='g=Content.gear.flat;e=profile.items.gear.equipped')
+    td
+      span.hint(popover-title=env.t('{{v.title}}'), popover-placement='right', popover=env.t('{{v.popover}}'), popover-trigger='mouseenter', style='margin-right:3px')
+        strong=env.t('{{v.title}}')
+      span
+        strong : {{profile._statsComputed.k}}
+    td
+      ul.list-unstyled
+        li(ng-show='profile.stats.lvl > 1')
+          span.hint(popover-title=env.t('levelBonus'), popover-trigger='mouseenter', popover-placement='top', popover=env.t('levelBonusText'))=env.t('level')
+          |: {{Math.ceil((Math.min(profile.stats.lvl - 1, 100)) / 2)}}&nbsp;
+        li(ng-show='Content.gear.flat[profile.items.equipped.weapon].k + g[e.armor].k + g[e.head].k + g[e.shield].k > 0')
```

Comment:
This line is different due to testing. I wanted to see if the error was related to the `ng-init`, or the k/v values. This didn't change the behavior, indicating that something is going wrong with the `k` values.